src/rawtoaces_util/image_converter.cpp

```diff
@@ -1180,21 +1232,24 @@ bool ImageConverter::configure(
     options["raw:Demosaic"]           = settings.demosaic_algorithm;
     options["raw:threshold"]          = settings.denoise_threshold;
 
+    const OIIO::TypeDesc float_vec4_type_desc(
+        OIIO::TypeDesc::FLOAT, OIIO::TypeDesc::AGGREGATE::VEC4 );
+    const OIIO::TypeDesc int_vec4_type_desc(
+        OIIO::TypeDesc::INT, OIIO::TypeDesc::AGGREGATE::VEC4 );
+    const OIIO::TypeDesc float_vec2_type_desc(
+        OIIO::TypeDesc::FLOAT, OIIO::TypeDesc::AGGREGATE::VEC2 );
```

Comment:
TODO Eithe rmove it up top outside all methods and use them in other methods or use inline in all the places.